venona/cmd/start.go

```diff
@@ -66,7 +75,7 @@ func init() {
 	viper.SetDefault("codefresh-host", defaultCodefreshHost)
 	viper.SetDefault("port", "8080")
 
-	startCmd.Flags().BoolVar(&startCmdOptions.verbose, "verbose", true, "Show more logs")
+	startCmd.Flags().BoolVar(&startCmdOptions.verbose, "verbose", false, "Show more logs")
```

Comment:
Now the default verbose is false, i think we need to update some log calls from `Debug` to `Info` across the app

Reply:
Changed a few important calls from Debug to Info